next.config.js

```diff
@@ -30,3 +38,18 @@ module.exports = {
     sentryDsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
   },
 };
+
+const SentryWebpackPluginOptions = {
+  // Additional config options for the Sentry Webpack plugin. Keep in mind that
+  // the following options are set automatically, and overriding them is not
+  // recommended:
+  //   release, url, org, project, authToken, configFile, stripPrefix,
+  //   urlPrefix, include, ignore
+  // For all available options, see:
+  // https://github.com/getsentry/sentry-webpack-plugin#options.
```

Comment:
I'd remove the comments in this file (above as well), they are just copied from the documentation